test/test_helper.rb

```diff
@@ -12,7 +12,7 @@ def add_car_to_cart
     create_cars(1)
     visit cars_path
 
-    click_link 'Add to my Garage'
+    click_button 'Drive'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.